cfgov/jinja2/v1/_includes/ask/share-links-spanish.html

```diff
@@ -1,3 +1,3 @@
-<a href="https://www.facebook.com/sharer.php?u={{ request.url }}{% if utm_campaign %}%3Futm_campaign={{ utm_campaign }}{% endif %}" class="share-facebook"><img src="{{ static('nemo/_/img/icon-facebook-retina.png') }}" alt="Share on Facebook" class="icon-20"></a>
-<a href="https://twitter.com/share?url={{ request.url|urlencode }}{% if utm_campaign %}%3Futm_campaign={{ utm_campaign }}{% endif %}&text={{ tweet_text if tweet_text else 'CFPB - Obtener Respuestas' }}&via=CFPB&lang=es" class="share-twitter"><img src="{{ static('nemo/_/img/icon-twitter-retina.png') }}" alt="Share on Twitter" class="icon-20"></a>
-<a href="https://api.addthis.com/oexchange/0.8/forward/email/offer?url={{ request.url|urlencode }}{% if utm_campaign %}%3Futm_campaign={{ utm_campaign }}{% endif %}&title={{  page.question if page.question else 'CFPB - Obtener Respuestas' }}&pubid=ra-4da354ee346886d2" class="share-email addthis_button_email"><img src="{{ static('nemo/_/img/icon-email-retina.png') }}" alt="Share via email" class="icon-20"></a>
+<a href="https://www.facebook.com/sharer.php?u={{ request.build_absolute_uri()|urlencode }}{% if utm_campaign %}%3Futm_campaign={{ utm_campaign }}{% endif %}" class="share-facebook"><img src="{{ static('nemo/_/img/icon-facebook-retina.png') }}" alt="Share on Facebook" class="icon-20"></a>
+<a href="https://twitter.com/share?url={{ request.build_absolute_uri()|urlencode }}{% if utm_campaign %}%3Futm_campaign={{ utm_campaign }}{% endif %}&text={{ tweet_text if tweet_text else 'CFPB - Obtener Respuestas' }}&via=CFPB&lang=es" class="share-twitter"><img src="{{ static('nemo/_/img/icon-twitter-retina.png') }}" alt="Share on Twitter" class="icon-20"></a>
+<a href="https://api.addthis.com/oexchange/0.8/forward/email/offer?url={{ request.build_absolute_uri()|urlencode }}{% if utm_campaign %}%3Futm_campaign={{ utm_campaign }}{% endif %}&title={{  page.question if page.question else 'CFPB - Obtener Respuestas' }}&pubid=ra-4da354ee346886d2" class="share-email addthis_button_email"><img src="{{ static('nemo/_/img/icon-email-retina.png') }}" alt="Share via email" class="icon-20"></a>
```

Comment:
TRIVIAL: Probably should place a space around the pipe, as is done in the social-media.html in this PR `request.build_absolute_uri() | urlencode`

Reply:
@anselmbradford thanks, will do.